server/functional-test/src/main/java/org/zanata/util/TestEventForScreenshotListener.java

```diff
@@ -95,18 +96,17 @@ private void createScreenshot(String ofType) {
 
                 Rectangle captureRectangle = new Rectangle(pos.x, pos.y, size.width, size.height);
 //                Rectangle captureRectangle = getScreenRectangle();
-
-                BufferedImage capture = new Robot().createScreenCapture(
-                        captureRectangle);
-                if (!ImageIO.write(capture, "png", screenshotFile)) {
-                    log.error("png writer not found for screenshot");
-                }
+                capture = new Robot().createScreenCapture(captureRectangle);
             } else {
                 File tempFile =
                         ((TakesScreenshot) driver).getScreenshotAs(OutputType.FILE);
```

Comment:
@djansen-redhat Instead of writing out a FILE and then reading the temp file off the disk, it would be better to capture as BYTES in memory. Plus I'm not sure the tempFile gets cleaned up.